tests/coalaTest.py

```diff
@@ -34,7 +34,7 @@ def test_coala(self):
             self.assertIn('This file has 1 lines.',
                           stdout,
                           'The output should report count as 1 lines')
-            self.assertIn('During execution of coala', stderr)
+            # self.assertIn('During execution of coala', stderr)
```

Comment:
@sils As described in https://github.com/coala/coala/pull/3338#issuecomment-293491125 , I first wanted to get some opinions about those asserts. If they have a useful purpose or if they were just introduced because there was some error output which did not really make sense but hey... before doing investigation better just accept and assert it ;)

I prefer just to remove those asserts, because those errors just arose from https://github.com/coala/coala/pull/3338/files#diff-f4c53753be8ec5d77fa199b20ac1caa4L165 and don't really make sense in the use cases of those tests.